packages/infra/lib/ihe-stack/ihe-gw-construct.ts

```diff
@@ -216,26 +222,29 @@ export default class IHEGatewayConstruct extends Construct {
       // don't create a new TG if this listener was already created on the same port
       if (existingListener) return;
       const targetGroupId = `${id}-TG-${port}`;
-      service.registerLoadBalancerTargets({
-        containerName,
-        containerPort: port,
-        protocol: ecs.Protocol.TCP,
-        newTargetGroupId: targetGroupId,
-        listener: ecs.ListenerConfig.applicationListener(listener, {
-          protocol: ApplicationProtocol.HTTP,
-          port,
-          targetGroupName: targetGroupId,
-          healthCheck: {
-            ...healthCheck,
-            ...(healthcheckInterval ? { interval: healthcheckInterval } : undefined),
-          },
-          // See for details: https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-target-groups.html#deregistration-delay
-          deregistrationDelay: Duration.minutes(10),
-        }),
+
+      const target = listener.addTargets(targetGroupId, {
+        protocol: ApplicationProtocol.HTTP,
+        port,
+        targetGroupName: targetGroupId,
+        targets: [
+          service.loadBalancerTarget({
+            containerName,
+            containerPort: port,
```

Comment:
is `protocol: ecs.Protocol.TCP,` not required to be specified within `loadBalancerTarget`?

Reply:
Not required, this compiles and I branched to staging and it worked. 👍